models/data_maps.go

```diff
@@ -283,3 +288,12 @@ func insertsIntoDataMapsTable(columnsName string, values string) error {
 	rawQuery := fmt.Sprintf("INSERT INTO %s (%s) VALUES %s", DataMapTableName, columnsName, values)
 	return DB.RawQuery(rawQuery).All(&[]DataMap{})
 }
+
+func GetDataMapByGenesisHashAndChunkIdx(genesisHash string, chunkIdx int) (dataMaps []DataMap, err error) {
+	dataMaps = []DataMap{}
+	err = DB.Where("genesis_hash = ?",
+		genesisHash).Where("chunk_idx = ?", chunkIdx).All(&dataMaps)
+
+	return dataMaps, err
```

Comment:
nit: just do: return
since you already have dataMaps and err as declaration. 
Also, it is more readability do this:

func GetDataMapByGenesisHashAndChunkIdx(genesisHash string, chunkIdx int) ([]DataMap, error) {

Reply:
Okay I'll change this.  Since I'm getting rid of the declared return values I'll still have to explicitly return dataMaps and err.